config/locales/cs.yml

```
@@ -8,38 +8,38 @@
 #  8. If a value has Markdown syntax characters in it, please keep those in your translation.
 #  9. Markdown examples: [link text](url), **bold**, _italics_, %{dynamic variable}, etc.
 # 10. If a value has <b>HTML</b> syntax characters in it, please keep those in your translation.
-cs:
+en:
```

This was probably a copy/paste accident, but this root node needs to match the file name.

```suggestion
cs:
```